src/worker/worker/tests/test_collector.py

```diff
@@ -30,14 +30,84 @@ def news_item_upload_mock(requests_mock):
 
 
 @pytest.fixture
-def collecotors_mock(osint_source_update_mock, news_item_upload_mock):
+def collectors_mock(osint_source_update_mock, news_item_upload_mock):
     pass
 
 
+def file_loader(filename):
+    with open(filename, "r") as f:
+        return f.read()
+
+
+@pytest.fixture
+def rss_collector_mock(requests_mock, collectors_mock):
+    from testdata import rss_collector_url, rss_collector_fav_icon_url, rss_collector_targets
+
+    requests_mock.get(rss_collector_targets[0], json={})
+    requests_mock.get(rss_collector_targets[1], json={})
+    requests_mock.get(rss_collector_targets[2], json={})
+    requests_mock.get(rss_collector_fav_icon_url, json={})
+    requests_mock.get(rss_collector_url, text=file_loader("test_rss_feed.xml"))
```

Comment:
Could be changed to
`    requests_mock.get(rss_collector_url, body=open("test_rss_feed.xml", "r"))`